### ISSUE TITLE
Problem with album thumbnail

### ISSUE THREAD

**Issue by rosimittermeier** [Philio/bootstrapdefault#18]
I have piwigo 2.7.4. And the bootstrap-theme is activated and works generally good. but if I change the thumbnail of an album, the theme saves the chage but does not show any thumb (neither the old nor the new one).

**Comment by Philio** [comment]
Hi, Could you provide a URL that demonstrates this behaviour?

**Comment by rosimittermeier** [comment]
yes, http://galerie.andreasvieth.de I chose successfully one of the pictures in the gallery as thumb. (It is the picture No. P1020053) But as you my see the picture is not shown. I you enter the gallery the picture is shown. It is on the server. It is successfully synchronized in piwigo. But chosen as the thumb of the gallery it's not shown. If I choose the standard themes (elegant, clear) everything is fine.

**Comment by Philio** [comment]
Did you intentionally close the issue?

**Comment by rosimittermeier** [comment]
No. I only commented. Maybe I made something wrong.

**Comment by Philio** [comment]
Ok well for some reason the custom thumbnail generation is failing. I'm afraid I'm not sure what could cause that but @plegall might have some insight.

**Comment by rosimittermeier** [comment]
Ok, thanks!!!!! I will ask him!

**Comment by Philio** [comment]
Hopefully he'll see that I tagged him and reply here!
